python/src/PythonRandomVector.cxx

```diff
@@ -249,6 +249,67 @@ Bool PythonRandomVector::isEvent() const
   }
 }
 
+/* Parameter accessor */
+Point PythonRandomVector::getParameter() const
+{
+  if (PyObject_HasAttrString(pyObj_, const_cast<char *>("getParameter")))
+  {
+    ScopedPyObjectPointer callResult(PyObject_CallMethod( pyObj_,
+                                     const_cast<char *>( "getParameter" ),
+                                     const_cast<char *>( "()" ) ));
+    if (callResult.isNull())
+    {
+      handleException();
+    }
+    Point parameter(convert< _PySequence_, Point >(callResult.get()));
+    return parameter;
+  }
+  else
+  {
+    // DistributionImplementation::getParameter throws
```

Comment:
Why do you mention DistributionImplementation in the comment?

Reply:
I don't know! I did not write these lines, I simply copied them from [PythonDistribution.cxx](https://github.com/josephmure/openturns/blob/03980409340dee0d2e7f56a1821f97cb74dac740/python/src/PythonDistribution.cxx#L929).

Reply:
@jschueller authored the lines back in PR #612

Reply:
When you copy/paste code, you also have to check comments ;-)